content/docs/deployment/csminstallationwizard/src/static/js/tests/generate-yaml.test.js

```diff
@@ -101,6 +101,16 @@ describe("GIVEN setValues function", () => {
             <input type="checkbox" id="leader-election">
             <input type="text" id="authorization-proxy-host" value="">
             <input type="text" id="taint" value="node-role.kubernetes.io/control-plane">
+            <input type="text" id="target-cluster-id" value="">
+            <input type="text" id="transport-protocol" value="">
+            <input type="checkbox" id="iscsichap">
+            <input type="checkbox" id="observability-operator">
+            <input type="checkbox" id="observability-operator-metrics">
+            <input type="checkbox" id="vobservability-operator-topology">
```

Comment:
There is a typo in the id "vobservability-operator-topology

Reply:
fixed.